src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -0,0 +1,217 @@
+﻿namespace Nancy.Json.Simple
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using Nancy.Extensions;
+
+    internal class NancySerializationStrategy : PocoJsonSerializerStrategy

```

Comment:
Only issue with that, it might not be an issue but users could then call Nancy.Simple.SimpleJson.Deserialize from anywhere in their app